.eslintrc.js

```diff
@@ -1,11 +1,17 @@
 module.exports = {
   extends: ['algolia', 'algolia/react'],
   rules: {
-    'react/prop-types': 0,
-    'no-param-reassign': 0,
     'eslint-comments/disable-enable-pair': 0,
+    'import/order': 0,
+    'no-param-reassign': 0,
+    'react/function-component-definition': 0,
+    'react/jsx-boolean-value': 0,
+    'react/jsx-filename-extension': 0,
+    'react/jsx-handler-names': 0,
+    'react/jsx-no-bind': 0,
```

Comment:
It's hard to understand why rules are disabled when you look at an `.eslintrc.js` file later on. Could you add comments so we keep a history of those decisions? This will also help users.

Reply:
Added comments in d2f59bc.